src/app/top-navigation/top-navigation.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
you are using " allover the project and ' in this file. Fix that. You have to use single code-style.

Reply:
default style for new component, i never open it before
done